### PR TITLE
Update SAML instructions for dynamic SSO URLs

### PR DESCRIPTION
**Pull request by IanHakes-okta**
## Description:
- **What's changed?** 
Added instruction that SSO URL fields can now accept non-static URL values.
Fixed typo.
- **Is this PR related to a Monolith release?** 
Yes. 2021.04.0

### Resolves:

* [OKTA-380611](https://oktainc.atlassian.net/browse/OKTA-380611)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/saml2/submit.md [hunk 1]
```diff
@@ -6,9 +6,9 @@
...
 
-* **Is your app capable of requesting other SSO URLs?** &mdash; Select this option to configure support for multiple ACS URLs where the SAML Response can be sent. Specify an index or URL to uniquely identify each ACS URL endpoint. If an **AuthnRequest** message does not specify an index or URL, the SAML Response is sent to the default ACS URL specified above. Enter the SSO URLs and an index value for any other nodes.
+* **Is your app capable of requesting other SSO URLs?** &mdash; Select this option to configure support for multiple ACS URLs where the SAML Response can be sent. Specify an index or URL to uniquely identify each ACS URL endpoint. If an **AuthnRequest** message does not specify an index or URL, the SAML Response is sent to the default ACS URL specified above. If **Yes**, enter the SSO URLs and an index value for any other nodes. You can specify variables in the SSO URL field to construct non-static URLs. For example, https://`${app.variableName}`.okta.com`.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/saml2/submit.md:11]:
```suggestion
* **Is your app capable of requesting other SSO URLs?** &mdash; Select this option to configure support for multiple ACS URLs where the SAML Response can be sent. Specify an index or URL to uniquely identify each ACS URL endpoint. If an **AuthnRequest** message does not specify an index or URL, the SAML Response is sent to the default ACS URL specified above. If **Yes**, enter the SSO URLs and an index value for any other nodes. You can specify variables in the SSO URL field to construct non-static URLs. For example, `https://`${app.variableName}.okta.com`.
```

Comment (joeysokolon-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/saml2/submit.md:11]:
```suggestion
* **Is your app capable of requesting other SSO URLs?** &mdash; Select this option to configure support for multiple ACS URLs where the SAML Response can be sent.  If **Yes** specify an index or URL to uniquely identify each ACS URL endpoint. If an **AuthnRequest** message does not specify an index or URL, the SAML Response is sent to the default ACS URL specified above. Enter the SSO URLs and an index value for any other nodes. You can specify variables in the SSO URL field to construct non-static URLs. For example, https://`${app.variableName}`.okta.com`.
```

**Review by joeysokolon-okta — APPROVED** [review]
LGTM

File: packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/saml2/submit.md (hunk 1 above)
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/saml2/submit.md]: Change made

**Pull request merged**